runtime/resolvers/metrics_sql.go

```diff
@@ -47,7 +47,12 @@ func newMetricsSQL(ctx context.Context, opts *runtime.ResolverOptions) (runtime.
 		return nil, err
 	}
 
-	compiler := metricssqlparser.New(ctrl, opts.InstanceID, opts.UserAttributes)
+	sqlArgs := &sqlArgs{}
+	if err := mapstructure.Decode(opts.Args, sqlArgs); err != nil {
+		return nil, err
+	}
+
+	compiler := metricssqlparser.New(ctrl, opts.InstanceID, opts.UserAttributes, sqlArgs.Priority)
```

Comment:
A little confusing to overload `sqlArgs` – maybe add an explicit `metricsSQLArgs` type?

Reply:
Sure. Fixed.